tasks/github_tasks.py

```diff
@@ -112,7 +112,7 @@ def find_packages_without_owner(owners, folder):
     return pkg_without_owners
 
 
-def get_code_owners(root_folder):
+def _get_code_owners(root_folder):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Are `_` prefixed functions in Python a known pattern? Or otherwise, what does `_` signify?

Reply:
yes it's to mean a private method (but it's just a convention)